Libraries/LibGUI/SortingProxyModel.h

```diff
@@ -55,9 +55,12 @@ class SortingProxyModel final : public Model {
 private:
     explicit SortingProxyModel(NonnullRefPtr<Model>&&);
 
+    virtual void on_model_update(unsigned) override;
```

Comment:
It took a moment for me to understand that this takes flags, so it should probably be explicit
```suggestion
    virtual void on_model_update(unsigned flags) override;
```